File: seongkyu-lim/Programmers/step1/Python/제일작은수제거하기.py
def solution(arr):
    '''
    answer = arr
    if len(answer) > 1 :
        small = arr[0]
        for i in range(1,len(arr)) :
            if arr[i]<small :
                small = arr[i]
        answer.remove(small)
    else :
        answer[0] = -1
    '''
    if len(arr) > 1:
        arr.remove(min(arr))
    else:
        arr[0] = -1
    answer = arr
    return answer

File: mmkser10/javascript/programmers/제일_작은_수_제거하기.js
function solution(arr) {
    var answer = [];
    let min = Math.min.apply(null, arr);
    for(let i = 0 ; i < arr.length ; i++){
        if(arr[i] !== min){
            answer.push(arr[i]);
        }
    }
    return answer.length === 0 ? [-1] : answer;
}

// another sol
function anotherSolutionA(arr) {
    arr.splice(arr.indexOf(Math.min(...arr)), 1);
    return arr.length < 1 ? [-1] : arr;
}